sql/moz-fx-data-marketing-prod/acoustic/contact_raw_v1/metadata.yaml

```diff
@@ -21,6 +21,7 @@ owners:
 scheduling:
   dag_name: bqetl_acoustic_contact_export
   date_partition_parameter: submission_date
+  retries: 0
```

Comment:
> Additionally, to avoid wasted cost by attempting to re-run the query (if the query is invalid, retrying is pointless)

An invalid query isn't the only way this could fail.  I'd suggest keeping the default retries, otherwise something like a transient BigQuery service issue could cause this to sit failed until someone manually reruns it.

Reply:
Honestly in this case I'm okay to then manually kick-start the task, I'll bump it up to `1` I feel like that's enough since this is not something I'd expect to see much.